app/controllers/phone/map.js

```diff
@@ -20,9 +20,8 @@ function handleMapClick(e) {
 }
 
 /* Right now Alloy doesn't add circles/polylines/polygons to map properly when they're created in Alloy
- * See https://jira.appcelerator.org/browse/ALOY-1608 for more info
```

Comment:
JIRA is dead! Long live JIRA archive! https://jira-archive.titaniumsdk.com/ALOY/ALOY-1608

Reply:
linked to archive